appveyor.yml

```diff
@@ -0,0 +1,40 @@
+image: Visual Studio 2017
+
+environment:
+  host: x86_64-pc-windows-msvc
+  matrix:
+    - platform: x86_64
+      target: x86_64-pc-windows-msvc
+      channel: nightly-2018-12-15
+    - platform: arm64
+      target: aarch64-pc-windows-msvc
+      channel: nightly-2018-12-15
+matrix:
+  allow_failures:
+    - platform: arm64
+
+install:
+    - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
+    - appveyor DownloadFile https://www.nasm.us/pub/nasm/releasebuilds/2.14.02/win64/nasm-2.14.02-win64.zip -FileName nasm.zip
+    - 7z e -y nasm.zip
+    - rustup-init -yv --default-toolchain %channel% --default-host %host%
+    - set PATH=%PATH%;%USERPROFILE%\.cargo\bin;%APPVEYOR_BUILD_FOLDER%
+    - rustc -vV
+    - cargo -vV
+    - rustup target add %target%
+
+build_script:
+    - git submodule update --init
+    - cargo build --release --target=%target%
+    
+# Temporary search to find artifacts
+after_build:
+    - dir *.exe /s /b | find ^"temp^" /v /i | findstr /e .exe
```

Comment:
crav1e right now should just produce a library.

Reply:
Any idea how the libary is called/named?

Reply:
either it is `librav1e.lib` or `librav1e.a`, but I never tried crav1e on windows...